R/create_hazard_matrix_agetime.R

```diff
@@ -76,7 +76,7 @@ create_hazard_matrix_agetime <- function(dat,
   ## Matrix for 3D hazard function if strat not NULL
   if (!is.null(strat)) {
 
-    ## Integration matrix for cumululative hazard
+    ## Integration matrix for cumulative hazard
     cols <- unlist(apply(dat, 1, function(x) {
```

Comment:
Might be worth looking at `vapply` for this. `apply` does conversion from dat to matrix and then it converts output to a vector so it can cause some unexpected behaviour sometimes. vapply tends to be much more predictable
e.g.
```
vapply(seq_len(nrow(dat)), function(row_number) { do_stuff }, numeric(1))
```